src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -147,12 +147,15 @@ def run_pytest(
     config: Optional[str] = None,
     force: bool = False,
 ) -> TestResult:
+    # For some reason we need to explicitly request setuptools to get pytest-html to work.
```

Comment:
This is an ~ancient version of pytest-html that has a [real dependency](https://github.com/pytest-dev/pytest-html/blob/v1.22.1/pytest_html/plugin.py#L13) that it [does not declare](https://github.com/pytest-dev/pytest-html/blob/v1.22.1/setup.py#L15). Modern versions (e.g.: 3.1.1) don't appear to use pkg_resources.

Reply:
Alas, that is the most recent version that is compatible with the ancient pytest we pin to. I will update the comment.